terraform/cp4ba/variables.tf

```diff
@@ -70,7 +70,7 @@ variable "owner" {
 
 variable "cluster_config_path" {
   default     = "./.kube/config"
```

Comment:
Why did you remove the comment regarding Schematics?  This is still necessary as far as I know.

Reply:
Done.
```
  description = "directory to store the kubeconfig file. If running on Schematics, use `/tmp/.schematics/.kube/config`"
```